server/database/config.js

```diff
@@ -12,18 +12,33 @@ var orm = new Sequelize(dbconfig.database, dbconfig.username, dbconfig.password,
 
 var Messages = orm.define('messages', {
   message: Sequelize.TEXT,
-  name: Sequelize.STRING
+  name: Sequelize.STRING,
+  groupID: Sequelize.STRING
 });
 
 var Users = orm.define('users', {
-  name: Sequelize.STRING
+  name: Sequelize.STRING,
+  password: Sequelize.STRING,
+  groupID: Sequelize.STRING
+});
+
+var Groups = orm.define('groups', {
+  groupID: Sequelize.STRING
```

Comment:
I have a feeling that `groupID` was intended to be `name` because it was set as a string.